src/components/dev-hub/project-sign-up-form.js

```diff
@@ -56,15 +56,15 @@ const Form = React.memo(({ setSuccess, success }) => {
     const [email, setEmail] = useState('');
     const [projectDescription, setProjectDescription] = useState('');
     const [canSubmit, setCanSubmit] = useState(true);
-    const handleSubmit = e => {
+    const handleSubmit = async e => {
         e.preventDefault();
         setCanSubmit(false);
         const obj = {
             name,
             email,
             projectDescription,
         };
-        authenticate();
+        await authenticate();
         const callback = hasSuccess => {
```

Comment:
Hmm just tried on chrome incognito and am still getting the submission failed. I looked in the network tab and the stitch calls looked fine, not sure what is going on here?

Reply:
Well I re-started the development server and that seemed to work! It takes a second though, should we have some kind of loading state?

Reply:
hmm good point. maybe the authentication should happen on mount instead then ?

Reply:
I think it might have been something on my end?

Reply:
oo i thought I saw a comment about a loadign state

Reply:
Yeah I was trying to debug a bit but then restarting my environment worked so I updated that comment!

Reply:
I think this way may be better anyway, because once you pointed that out, I did notice the delay in seeing the loading state. What do you think?